docs/troubleshooting.md

```diff
@@ -35,3 +36,28 @@ using Minishift.
   systemctl start virtlogd
   systemctl enable virtlogd
   ```
+
+
+## XHYVE drivers
+
+- Minishift failed to start because of ```Error: could not create vmnet interface, permission denied or no entitlement```.
+
+    The problem is likely to be that the xhyve driver is not able to clean up vmnet when a VM is removed.
+
```

Comment:
not able to clean up vmnet>not able to clean vmnet/not able to clean the `vmnet.framework`
Do we need back ticks for vmnet.framework on line 45 and 48? either ways let us keep it consistent everywhere.